pkg/plugins/resources/gitlab/client/main.go

```diff
@@ -13,7 +12,7 @@ import (
 // Spec defines a specification for a "Gitlab" resource
 // parsed from an updatecli manifest file
 type Spec struct {
-	// [S][C][T] URL specifies the default github url in case of Gitlab enterprise
+	// [S][C][T] URL specifies the default Gitlab url in case of Gitlab enterprise
```

Comment:
```suggestion
	// [S][C][T] URL specifies the default Gitlab Url
```

There is no difference between GitLab.com or any self-hosted instances. I think the "enterprise" bit was also left over from GitHub.

Reply:
Thanks for cleaning this up